sources/views/grid_view.js

```diff
@@ -1,47 +1,52 @@
 import {JetView} from "webix-jet";
 import {textValidation} from "models/validate_function";
 import {scrollToLastAddedElemen} from "models/scroll_function";
+
 export default class GridView extends JetView {
 	constructor(app, data){
 		super(app);
 		this.data = data;
 	}
+
 	config(){
-		var add_new_button = {
+		const _ = this.app.getService("locale")._;
+
+		const add_new_button = {
 			view:"button",
-			value:"Add new",
+			value:_("Add new"),
 			css:"webix_primary",
 			click:()=>{
-				webix.message({text:"New element was added", expire:350});
-				this.grid.add({Name:""});
+				webix.message({text:_("New element was added"), expire:350});
+				this.grid.add({id:0, value:"New"});
```

Comment:
do not use ids while adding new items (they can be non unique). just omit them - the view will generate the unique id automatically

do not use `id:0` - it is forbidden for webix views, as it means no id

Reply:
ok